src/sage/groups/perm_gps/permgroup.py

```diff
@@ -278,12 +280,31 @@ def PermutationGroup(gens=None, *args, **kwds):
 
     - a permutation group
 
+    .. NOTE::
+
+        If the domain is not specified, and all generators act on
+        integers, the domain is the smallest interval of integers
```

Comment:
be more precise about the type of integers. `ZZ` ? `int` ?

Reply:
I'd  prefer a full detail, using Sphinx `:class:` tag to specify the type completely.